distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -1379,6 +1379,483 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:Intersects>
+                   <ogc:PropertyName>ows:BoundingBox</ogc:PropertyName>
+                   <gml:Polygon srsName="EPSG:4326">
+                       <gml:exterior>
+                           <gml:LinearRing>
+                               <gml:pos>69.31 33.2</gml:pos>
+                               <gml:pos>70.73 33.22</gml:pos>
+                               <gml:pos>69.66 34.29</gml:pos>
+                               <gml:pos>69.31 33.2</gml:pos>
+                           </gml:LinearRing>
+                       </gml:exterior>
+                   </gml:Polygon>
+               </ogc:Intersects>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+A query for records that contain a line in its bounding box.
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:Contains>
+                   <ogc:PropertyName>ows:BoundingBox</ogc:PropertyName>
+                   <gml:LineString srsName="EPSG:4326">
+                       <ns2:pos>25 31</ns2:pos>
+                       <ns2:pos>27 32 </ns2:pos>
+                   </gml:LineString>
+               </ogc:Contains>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+===== `XPath Operation Example`
+A xpath query for records with a xpath property with a value like "TextToMatch".
```

Comment:
It might be worth mentioning that the xpath expression is evaluated against the internal document structure instead of the outputSchema structure.